### PR TITLE
Rewrite parts of code of conduct

### PR DESCRIPTION
**Pull request by dafeda**
**Issue**
No issue

**Approach**
In large part a rewrite to more "active" English. 
Not sure if "active" is the right way to describe this, but typically instead of "Equinor maintained projects", I propose writing "projects maintained by Equinor".

A few paragraphs are rewritten more completely.

I also propose having one sentence per line for easier diffing.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,76 +1,53 @@
 # Code of conduct for open source projects in Equinor
+
 In Equinor, [how we deliver is as important as what we deliver](https://www.equinor.com/en/careers/our-culture.html).
 
-This principle applies to all work done in Equinor maintained projects as well
-as for any Equinor contribution to other projects within the open source
-community.
+This principle applies to all work done in projects maintained by Equinor, as well as for any contributions by Equinor to other projects within the open-source community.
```
Comment (jondequinor) [inline CODE_OF_CONDUCT.md:5]: I think you've flattened the sentence, which allows for easier reading.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,76 +1,53 @@
...
-it is a matter of repeatedly failing to take actions with respect to a mandated
-decision.
+To encourage members of the community to file reports, all reports are kept confidential unless you choose to disclose it yourself.
 
-For Equinor maintained projects, only the Equinor open source team has the
-mandate for making a corrective action against individuals based on the code of
-conduct. These decisions are never made within each project alone. As a last
-resort, a potential consequence may be exclusion from participation in that
-particular community.
+Consequences of violations are measured after the severity of the issue, or if it is a matter of repeatedly failing to take actions with respect to a mandated decision.
```
Comment (jondequinor) [inline CODE_OF_CONDUCT.md:39]: Maybe simply "Consequences of violations reflect the severity of the issue, and/or repeated failure to take action following mandated decisions."

Comment (dafeda) [reply CODE_OF_CONDUCT.md:39]: That looks good!

**Comment by jondequinor** [comment]
jenkins add to whitelist

**Comment by sondreso** [comment]
Nice! I suggest also making a PR with these changes here: https://github.com/equinor/it-professional-network/blob/master/doc/open_source/code_of_conduct/CODE_OF_CONDUCT.md

**Comment by dafeda** [comment]
> Nice! I suggest also making a PR with these changes here: https://github.com/equinor/it-professional-network/blob/master/doc/open_source/code_of_conduct/CODE_OF_CONDUCT.md

I'll create a PR once this is merged.

**Review by jondequinor — APPROVED** [review]
LGTM! Thanks for improving this.

**Comment by dafeda** [comment]
jenkins test this please

**Pull request merged**